hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/upgrade/TestOMVersionManager.java

```diff
@@ -54,11 +61,39 @@ public void testOMLayoutVersionManager() throws IOException {
     assertEquals(2, omVersionManager.getMetadataLayoutVersion());
   }
 
+  @Test
+  public void testOMLayoutVersionManagerInitError() {
+    OMStorage omStorage = mock(OMStorage.class);
+    when(omStorage.getLayoutVersion()).thenReturn(
+        OMLayoutFeature.values()[OMLayoutFeature.values().length - 1]
+            .layoutVersion() + 1);
+    try {
+      OMLayoutVersionManager.initialize(omStorage);
+      Assert.fail();
+    } catch (OMException ex) {
+      assertEquals(NOT_SUPPORTED_OPERATION, ex.getResult());
+    }
+  }
+
+  @Test
+  public void testOMLayoutVersionManagerReset() throws IOException {
```

Comment:
Just curious about this scenario, when will we invoke OMLayoutVersionManager#resetLayoutVersionManager call.

Reply:
@linyiqun That was done just for testing purposes. In the future, we may move away from "static" Version Manager class, and that time we can remove this.